project/DistributionPackage.scala

```diff
@@ -375,9 +375,6 @@ object DistributionPackage {
       all.add("--disable-private-check")
     }
     pb.command(all)
-    workingDirectory
```

Comment:
I don't understand why we [were changing CWD](https://github.com/enso-org/enso/pull/12901/files#r2072653116) before executing `enso`, @radeusgd, @AdRiley, @Akirathan?

Better to avoid it and make sure that commands like:
```
sbt:enso> runEngineDistribution --docs api --in-project distribution/lib/Standard/Tableau/0.0.0-dev
```
find the relative paths without major surprises.

Reply:
I was using the explicit working directory override only in one case - when using `--new`. This option creates a new project in CWD so I needed to change it to create the project in desired directory.

Reply:
If you think it was wrong, then please do remove `.orElse(projectPath.map(new File(_).getParentFile))` but please keep the `workingDirectory.foreach(...)` part.

It is used in only _one_ scenario as described above with `--new`. Otherwise you will probably break the CI Lint check. Unless you have some different plan on how to make `--new` create the project in the right place.

Reply:
- true, the Lint check is broken: https://github.com/enso-org/enso/actions/runs/14833591482/job/41640039044#step:10:1368
- e4991e7bf6dedf82cef06f0fb771b598f44366b1 is my fix
- let's see if it fixes the Link check CI (locally it seems to do the job)

Reply:
Ok, sounds good

Reply:
But then the `workingDirectory` parameter must be removed from `runEnginePackage`.

It is not used anymore, so keeping it there is only confusing.

Reply:
Being removed in https://github.com/enso-org/enso/pull/13021/commits/702d63f6a681e0e4ed8017db80787eb3a4545767